src/muc_light/mod_muc_light_room_config.erl

```diff
@@ -73,7 +73,7 @@
 %%====================================================================
 
 -spec schema_from_definition(UserDefinedSchema :: user_defined_schema()) -> schema().
-schema_from_definition(UserDefinedSchema) ->
+schema_from_definition(UserDefinedSchema) when is_list(UserDefinedSchema) ->
```

Comment:
Why this numb change? `lists:foldl` will already check that it is a list, and this function should then be on the stacktrace. Did that go wrong somewhere?

Reply:
It makes stacktraces better, failing in foldl is less pretty.
Function clause on schema_from_definition is better.

(it's here because test cases use that function).